mappings/net/minecraft/test/TestFailureLogger.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_5623 net/minecraft/test/TestFailureLogger
```

Comment:
Maybe `TestResultLogger` when it reports passed tests as well?

Reply:
It passes it specifically only to the TestFailedReporter or whatever it's called (I'm on my phone right now, can't easily check) which only actually reports fails and does nothing with passes.